pyproject.toml

```diff
@@ -407,6 +265,147 @@ ignore = [  # NOTE: non-permeanent exclusions should be added to `.ruff.toml` in
 [tool.ruff.pydocstyle]
 convention = "numpy"
 
+[tool.pytest.ini_options]
```

Comment:
This I don't understand; either be alphabetical or not - why is this moved past `tools.ruff`? Certainly, `pytest` is far more important to see than `ruff`.